src/noc/containers/noc-affirmation/noc-affirmation.component.html

```diff
@@ -4,9 +4,9 @@
     <fieldset class="govuk-fieldset" aria-describedby="sign-affirmation">
       <p>You should tick to 'sign' when:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>You’re satisfied that all the information is accurate and matches what is written on the case</li>
-        <li>You have served notice of this change on every party to the claim (including the former legal
-          representative)
+        <li>You're satisfied that all these details are accurate and match what is written on the case</li>
```

Comment:
I see you have kept "You're" in on this line - I assume they didn't want to change this to "You are" like you have done with "you've"

Reply:
Good spot @paulhowes-HMCTS - however, the above wording was what was in the screenshot and they didn't raise any observations on this. From Jamie Mistry's comments:
> Hi....the wording in the screenshot is the one that has been agreed with Steve C and the rules committee

Reply:
Thanks Daniel. I thought as much. Already approved so resolving this conversation.